extension/CHANGELOG.md

```diff
@@ -1,8 +1,6 @@
 # Change Log
 
-## 0.9.0
-- Support for tracking channels per repository/extension.
-
 ## Unreleased
 
-- Initial release
\ No newline at end of file
+- Initial release
```

Comment:
I just pushed an update that makes 0.9.0-beta.0 the initial release, so you'll probably need to merge that into your branch and add a new "unreleased" section for this update.